### ISSUE TITLE
[ SUGGESTION] Being able to see CPU and GPU usage

### ISSUE THREAD

**Issue by arBorghi** [cominixo/BetterF3#113]
I love this mod but I feel that it lacks a basic feature: seeing the CPU and GPU usage

**Comment by Catomax262** [comment]
I think you can by downloading fpsdisplay

**Comment by djmrFunnyMan** [comment]
I agree. This is even present in the vanilla F3 menu so idk why this mod doesn't have it
![image](https://user-images.githubusercontent.com/68905776/223847392-42d64748-faad-43a9-a580-f86976e69656.png)
